### PR TITLE
updated wording around findings retention

### PR DESCRIPTION
**Pull request by s-santillan**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [ ] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team

### CODE REVIEW

File: docs/semgrep-code/findings.md [hunk 1]
```diff
@@ -145,12 +145,13 @@ To **add notes** to the activity history of a finding:
 
 ## Data retention
 
-The retention period for findings is **5 years**.
+Semgrep, Inc. retains findings data as long as an account remains active. We automatically securely destroy data within **90 days of contract termination** for our **Enterprise** customers.
```
Comment (khorne3) [inline docs/semgrep-code/findings.md:148]:
```suggestion
Semgrep, Inc. retains findings data as long as an account remains active. We automatically securely destroy data within **90 days of contract termination** for our **Enterprise** customers.
```

I'm not which should stay (automatically or securely) but I think there are too many adverbs in this sentence

Comment (khorne3) [reply docs/semgrep-code/findings.md:148]: Also, am I opening up a whole 'nother can of worms by asking what defines an active account?

**Review by khorne3 — APPROVED** [review]

File: docs/semgrep-code/findings.md (hunk 1 above)
Comment (s-santillan) [reply docs/semgrep-code/findings.md:148]: Hi @werrett ! Katie has a good point about "active account" - I don't think we have an inactive state?

Comment (werrett) [reply docs/semgrep-code/findings.md:148]: Hey both, we don't have an inactive state. So today we basically don't delete data but I think we want to retain the ability to do that. So 'active' is a good, intentionally vague, answer :)

Comment (s-santillan) [reply docs/semgrep-code/findings.md:148]: Got it. Thank you @werrett!

Comment (werrett) [reply docs/semgrep-code/findings.md:148]: I also removed the 'automatically' bit, as its not currently true.

**Review by werrett — APPROVED** [review]
LGTM! Thanks

**Pull request merged**